src/sysjson.go

```diff
@@ -39,9 +50,13 @@ func main() {
 
 func statsHandler(w http.ResponseWriter, r *http.Request) {
 	resp := map[string]interface{}{}
-
-	loadModules(resp, r.URL.Query().Get("modules"))
-
+	
+	if len(*from) == 0 || (len(*from) > 0 && strings.HasPrefix(r.RemoteAddr, *from)) {
```

Comment:
I worry that this may be confusing, since checking if an IP address has a prefix isn't typical address checking notation. What I mean by this is if I were to whitelist an IP address for sys.json to accept requests from, I would expect to supply something like "10.0.0.0/8 and not "10.".